spec/support/chunks.rb

```diff
@@ -3,7 +3,7 @@
 module Chunks
   def read_chunks(body)
     buffer = []
-    body.each { |chunk| buffer << chunk }
+    body.each { |chunk| buffer << chunk } # rubocop:disable Style/MapIntoArray
```

Comment:
I would rely on `rubocop --auto-gen-config`, we disable a lot of violations that way.